.github/workflows/cd.main.yml

```diff
@@ -0,0 +1,51 @@
+name: CD
+on:
+  push:
+    branches:
+      - main
+env:
+  ORGANIZATION_SLUG: templates
+  INSTRUQT_TOKEN: ${{ secrets.INSTRUQT_TOKEN }}
+jobs:
+  deploy-changes:
+    name: Deploy changes
+    runs-on: ubuntu-latest
```

Comment:
Ah I was hoping we could just run the action using our Docker image, but apparently that is not possible in Github actions :(